drive/modules/listener/constants.bal

```diff
@@ -107,6 +107,9 @@ const string UNABLE_TO_ENCODE = "Unable to encode value: ";
 // Error
 const string ERR_CHANNEL_REGISTRATION = "Unable to register new channel.";
 const string ERR_SCHEDULE = "Unable to schedule subscription renewal.";
+const string ERR_UNAUTHORIZED_WEBHOOK_CHANNEL = "Unauthorized WebHook callback channel";
+const string ERR_FILE_NOT_FOUND ="File not found";
+const string ERR_INVALID_FILE_OR_FOLDER ="Invalid folder or file !!";
```

Comment:
Cannot register watch to file or folder specified

Reply:
addressed in https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/55 & https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/56